From ae51e69d234b64c6a4be3883833dbab846526965 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Tue, 15 Mar 2022 20:20:48 +0200
Subject: [PATCH]  [SW-78698] gaudi3: update hash for key 1

Currently, the hash stored for gaudi3 for key1 are copied from gaudi2. This
patch updates key1 hash generated from gaudi3 production key 1.

Change-Id: I942f70809dd56ff235b9ee146e6a50e5d6edf9b1
---

diff --git a/arch/gaudi3/arc/security.c b/arch/gaudi3/arc/security.c
index 34bc845..582edbf 100644
--- a/arch/gaudi3/arc/security.c
+++ b/arch/gaudi3/arc/security.c
@@ -32,13 +32,12 @@
 	},
 	/* Production Key 1 */
 	{
-		/* TODO: Yet to receive production key pub key certificate */
-		0x47, 0x9b, 0x51, 0x5d, 0xf9, 0x8b, 0xfd, 0x3d,
-		0x38, 0x3f, 0x19, 0x51, 0x40, 0x62, 0xcb, 0x15,
-		0x2d, 0xae, 0x11, 0xe8, 0x1a, 0x91, 0x3e, 0x02,
-		0xa0, 0x80, 0x08, 0x9a, 0x62, 0x93, 0xb7, 0x65,
-		0x73, 0xe1, 0x6f, 0x86, 0x32, 0x16, 0xea, 0x37,
-		0x48, 0x19, 0xb8, 0xe0, 0xc3, 0x3b, 0x5a, 0xcf
+		0x60, 0x44, 0x16, 0xd8, 0x1b, 0xea, 0x2e, 0x66,
+		0x46, 0xf3, 0x01, 0x78, 0x2f, 0x3c, 0xf6, 0xfc,
+		0x88, 0x3d, 0xbb, 0x5d, 0x94, 0xe7, 0xd9, 0x94,
+		0x79, 0xea, 0x6e, 0xdc, 0x1d, 0xd8, 0x89, 0xe1,
+		0xa4, 0xeb, 0x28, 0x4a, 0x1b, 0x60, 0x87, 0xed,
+		0x58, 0x4a, 0xe1, 0xa0, 0xe9, 0x13, 0x6f, 0xb6
 	},
 	/* Production Key 2 */
 	{
